nodebuilder/fraud/module.go

```diff
@@ -15,11 +16,17 @@ func Module(tp node.Type) fx.Option {
 		return fx.Module(
 			"fraud",
 			fx.Provide(ServiceWithSyncer),
+			fx.Provide(func(service fraud.Service) fraud.Getter {
```

Comment:
make `baseComponents` for this since it's on all node types?

Reply:
done